src/api/status/Dockerfile

```diff
@@ -1,16 +1,63 @@
-FROM node:lts-alpine as base
+# Stage base #
+##############
+FROM node:16-alpine3.15 as base
 
-# https://snyk.io/blog/10-best-practices-to-containerize-nodejs-web-applications-with-docker/
-RUN apk add dumb-init
+# https://github.com/krallin/tini
+RUN apk --no-cache add tini=0.19.0-r0
```

Comment:
Rather than doing this, we can use `init: true` in `docker-compose`, which will do the same without installing an extra dependency.